src/app/fyle/dashboard/tasks/tasks.component.ts

```
@@ -4,7 +4,7 @@
 import { ActivatedRoute, Router } from '@angular/router';
 import { ModalController, PopoverController, RefresherEventDetail } from '@ionic/angular';
 import { Observable, BehaviorSubject, forkJoin, from, of, concat, combineLatest } from 'rxjs';
-import { finalize, map, shareReplay, switchMap } from 'rxjs/operators';
+import { catchError, finalize, map, shareReplay, switchMap } from 'rxjs/operators';
```

ESLint

[error] 'catchError' is defined but never used.

(@typescript-eslint/no-unused-vars)